tests/codegen-llvm/integer-overflow.rs

```diff
@@ -10,7 +10,7 @@ pub struct S1<'a> {
 // CHECK-LABEL: @slice_no_index_order
 #[no_mangle]
 pub fn slice_no_index_order<'a>(s: &'a mut S1, n: usize) -> &'a [u8] {
-    // CHECK-NOT: slice_index_order_fail
+    // CHECK-COUNT 1: slice_index_fail
```

Comment:
```suggestion
    // CHECK-COUNT-1: slice_index_fail
```
I'm not sure that this works, here and the other 3 instances in this PR.
https://llvm.org/docs/CommandGuide/FileCheck.html#the-check-count-directive

Reply:
You fixed the `CHECK-COUNT-<num>` in the other test file but not this one.